src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs

```diff
@@ -58,6 +58,13 @@
     :else (do (utils/log "Unknown workflow status: " status)
             failure-icon)))
 
+(defn icon-for-sub-status [wf-statuses]
+  (cond
+    (contains? wf-statuses "Failed") failure-icon
+    (contains? wf-statuses "Succeeded") success-icon
+    :else (do (utils/log "Unknown submission status")
+            nil)))
```

Comment:
is nil safe to return here? i don't want to display an icon if the submission isn't in either of the first two states